### PR TITLE
Videomaker: Update Videomaker navigation

### LINKED ISSUE #4656: videomaker: Navigation (if needed)
The Videomaker navigation looks like this:
Screenshot 2021-09-22 at 14 29 04

Screenshot 2021-09-22 at 14 29 18

We need to wait for https://github.com/WordPress/gutenberg/issues/34783 to merge before we can achieve all of this.

### PR DESCRIPTION
**Pull request by scruffian**
#### Changes proposed in this Pull Request:
Implements the idea in https://github.com/Automattic/themes/issues/4656#issuecomment-967120686

Screenshot 2021-11-18 at 17 29 33

#### Related issue(s):
#4656

### CODE REVIEW

**Comment by pbking** [comment]
The main link font sizes seem pretty far off from target.  Shouldn't they be much larger?

This change also has changes to the awards block patterns.  Perhaps branched from another branch with unnecessary changes?

**Comment by scruffian** [comment]
You're right, I updated the font sizes.

The menu items should also be vertically centered. This is pretty hard to do with the currrent implementation and would take significant reworking, so if we want to do that I suggest we do it in a followup PR.

**Comment by kjellr** [comment]
What is the best way to get a social navigation set up so that I can test that aspect of this? Using the customizer doesn't seem to do it: 

Screen Shot 2021-11-22 at 3 16 52 PM

... and doing this in the site editor gets weird results: 

Editor|Front End
---|---
Screen Shot 2021-11-22 at 3 19 06 PM|Screen Shot 2021-11-22 at 3 18 49 PM

**Comment by kjellr** [comment]
Oh wait, my mistake. I can get it working in the site editor (But not in the customizer? 🤷)

**Comment by kjellr** [comment]
I'm seeing very bizarre vertical spacing here (and in `trunk`) when I update this in the site editor:

Screen Shot 2021-11-22 at 3 24 07 PM



> The menu items should also be vertically centered. This is pretty hard to do with the currrent implementation and would take significant reworking, so if we want to do that I suggest we do it in a followup PR.

Not sure if this is viable, but this spacing bug would actually be solved if we adopted `justify-content: center` instead of `space-between` inside of the expanded menu:

Screen Shot 2021-11-22 at 3 29 11 PM

**Comment by scruffian** [comment]
Oh yeah nice one:
Screenshot 2021-11-24 at 13 51 44

**Comment by kjellr** [comment]
I'm seeing some weirdness when there are a lot of items in the menu. They start flowing (and scrolling) horizontally: 
Screen Shot 2021-11-24 at 9 01 48 AM
Screen Shot 2021-11-24 at 9 01 31 AM

**Comment by scruffian** [comment]
OK I worked out what was going on here - the social navigation was missing as you said above - you shouldn't need to add it from the site editor.

The problem is that if you have justify content center then when there's not many menu items it looks like this:
Screenshot 2021-11-24 at 17 31 57

The alternative with space-between looks like this:
Screenshot 2021-11-24 at 17 35 29

@beafialho @kjellr which do you prefer?

**Comment by kjellr** [comment]
I definitely prefer the top one. That looks right to me:

**Comment by scruffian** [comment]
In that case can we merge this? :)

File: videomaker/block-template-parts/header.html [hunk 1]
```diff
@@ -7,6 +7,6 @@
 		<!-- wp:site-tagline /-->
 	</div>
 	<!-- /wp:group -->
-	<!-- wp:navigation {"overlayMenu":"always","__unstableLocation":"primary","className":"is-style-blockbase-navigation-improved-responsive","layout":{"type":"flex","setCascadingProperties":true,"justifyContent":"right","orientation":"horizontal"}} /-->
+	<!-- wp:navigation {"overlayMenu":"always","__unstableLocation":"primary","__unstableSocialLinks":"social","className":"is-style-blockbase-navigation-improved-responsive","layout":{"type":"flex","setCascadingProperties":true,"justifyContent":"right","orientation":"horizontal"}} /-->
```
Comment (kjellr) [inline videomaker/block-template-parts/header.html:10]: What is this `__unstableSocialLinks` attribute about?

**Review by kjellr — APPROVED** [review]
Left one minor comment, but this is working on my end. 👍

File: videomaker/block-template-parts/header.html (hunk 1 above)
Comment (scruffian) [reply videomaker/block-template-parts/header.html:10]: This is how we show a social navigation inside a navigation block. It allows us to hide the social links inside the overlay, while still allowing menus to be updated via the customizer.

**Pull request merged**